src/plugins/files/server/file_client/create_es_file_client.ts

```diff
@@ -66,12 +67,18 @@ export function createEsFileClient(arg: CreateEsFileClientArgs): FileClient {
     maxSizeBytes,
     indexIsAlias,
   } = arg;
+
+  let hashes: Array<'sha1' | 'sha256' | 'sha512' | 'md5'> = ['sha1', 'sha256', 'sha512'];
+  if (getFips() !== 1) {
```

Comment:
Self Review: It seems that only fleet agents are using this function. Im not sure they are utilizing the `md5` hash at all, but I want to be sure. Thoughts on this change?

cc @jloleysens @kpollich

Reply:
This shouldn't be a problem for Fleet. We include a `sha256` hash for files uploaded from Elastic Agent (e.g. diagnostics bundles, Endpoint quarantined files). 

cc @paul-tavares for awareness since Endpoint relies on this file upload functionality pretty heavily.

Reply:
This change is fine with us as well. The Fleet Files client, which our team leverages for files that need to be sent to Agents or retrieved from agents, is only using `sha256`). Code reference:

https://github.com/elastic/kibana/blob/e9abdbd168f7076a719cb1ede3a91bad52de3b5f/x-pack/plugins/fleet/server/services/files/client_to_host.ts#L49-L56

Reply:
Excellent - thank you both!